rmgpy/rmg/pdep.py

```diff
@@ -277,6 +277,10 @@ def exploreIsomer(self, isomer):
             raise Exception('Attempted to explore isomer {0}, but that species not found in product channels.'.format(isomer))
 
         logging.info('Exploring isomer {0} in pressure-dependent network #{1:d}'.format(isomer, self.index))
+
+        for mol in isomer.molecule:
+            mol.update()
```

Comment:
What was the reason for adding this?

Reply:
Prior to this fix I had AtomTypeErrors in PDep jobs.
However, following you comment I now ran a few sulfur/nitrogen jobs with PDep, commenting-out these two lines; they all ran fine. So perhaps these lines are no longer necessary, but I can't be 100% sure.
We could remove this addition. If any atomType issue in PDep rises we'll see if this fixes it and could document it better.

Reply:
@alongd and I are not sure that the line is completely necessary, but have decided it probably cannot hurt to update the molecules. So, we have decided to leave this in.